.travis.yml

```diff
@@ -32,6 +32,18 @@ matrix:
         - DOCKERIMAGE=linux-anvil-aarch64
         - DOCKERTAG=latest
 
+    - os: linux
+      env:
+        - DOCKERIMAGE=linux-anvil-cuda
+        - DOCKERTAG=9.2
+        - CUDA_VER=9.2
+
+    - os: linux
+      env:
+        - DOCKERIMAGE=linux-anvil-cuda
+        - DOCKERTAG=10.0
+        - CUDA_VER=10.0
```

Comment:
Why build 2 versions?

Reply:
Are you meaning why CUDA 9.2 and CUDA 10.0 or are you referring specifically to the use of `DOCKERTAG` and `CUDA_VER`?

Reply:
> Are you meaning why CUDA 9.2 and CUDA 10.0 

This one

Reply:
This is a fair question. Though the question (and answer) isn't really too different from why we support multiple Python versions.

It's useful to have an older version around for large user groups that may upgrade a bit slower. Though it's also nice to have a newer version that is at the cutting edge in terms of features and performance for smaller groups that are able to easily upgrade.

The one difference with CUDA is just building on an older version and supplying that for all users (even on newer CUDA versions) can be a bit problematic for a few reasons. First it can introduce longer JIT compile times for users that are on the latest version. Second users that are able to upgrade quickly do so without very much hesitation in the GPU world. Third newer CUDA versions are often making better use of recent hardware.

As these issues and needs are discussed in greater detail in issue ( https://github.com/conda-forge/conda-forge.github.io/issues/687 ), I'll try to avoid rehashing them here.